src/components/pages/film/tabs.snap.test.js

```diff
@@ -16,8 +16,8 @@ const store = {
     activeTab: TabTypes.OVERVIEW,
   }
 };
-describe(`Tabs should render correctly`, () => {
-  it(`Tabs should render correctly if active tab is Overview`, () => {
+describe(`Tabs`, () => {
+  it(`renders correctly if active tab is Overview`, () => {
```

Comment:
просто показываю пример как бы я называл

```suggestion
describe(`Tabs component`, () => {
  it(`should render active Overview tab correctly`, () => {
```